src/internal/storage/fileset/storage.go

```diff
@@ -164,18 +230,117 @@ func (s *Storage) ComposeTx(tx *pachsql.Tx, ids []ID, ttl time.Duration) (*ID, e
 // CloneTx creates a new fileset, identical to the fileset at id, but with the specified ttl.
 // The ttl can be ignored by using track.NoTTL
 func (s *Storage) CloneTx(tx *pachsql.Tx, id ID, ttl time.Duration) (*ID, error) {
-	md, err := s.store.GetTx(tx, id)
+	return &id, nil
```

Comment:
If I wanted to spend more time on this, I'd create a table in postgres (original, cloned). I'd then traverse this for graphs to figure out if I could get the related total or diff commit metadata.